fields/types/list/ListColumn.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ItemsTableCell from '../../components/ItemsTableCell';
+import ItemsTableValue from '../../components/ItemsTableValue';
+import { plural } from '../../../admin/client/utils/string';
+
+var ListColumn = React.createClass({
```

Comment:
Honestly, copy + paste... I guess consistency with the rest of the Column classes?